Vue3

kylewalow opened the pull request.

kylewalow committed:
update plugin entry for vue3

kylewalow committed:
update package version

kylewalow committed:
Merge branch 'main' into vue3

strebl commented on package.json:
```diff
@@ -49,10 +49,10 @@
+    "rollup-plugin-vue": "^6.0.0",
+    "standard-version": "latest",
+    "vue": "^3.2.45",
+    "vue-template-compiler": "^2.6.11"
```
isch das wirklich die aktuellsti und richtigi verson?

kylewalow replied:
Ja das chunnt vomene upgrade latest

kylewalow committed:
fix unbluring of images loaded from cache

https://stackoverflow.com/a/73062650
ChatGPT: "It's worth noting that the load event will not fire if the image is already cached by the browser, it will only fire when the image is loaded from the server."

kylewalow committed:
update import path

kylewalow committed:
update package-lock

kylewalow committed:
drop aspect-ratio attribute on img

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-height

kylewalow committed:
update version patch
